packages/preview/modern-unito-thesis/0.1.0/lib.typ

```diff
@@ -0,0 +1,326 @@
+  set enum(indent: 10pt, body-indent: 9pt)
+  set list(indent: 10pt, body-indent: 9pt, marker: ([•], [--]))
+
+  // Configure headings
+  set heading(numbering: "1.a.I")
+  show heading.where(level: 1): it => {
+    if it.body not in ([References], [Riferimenti]) {
+      block(width: 100%, height: 20%)[
+        #set align(center + horizon)
+        #set text(1.3em, weight: "bold")
+        #smallcaps(it)
+      ]
+    } else {
+      block(width: 100%, height: 10%)[
+        #set align(center + horizon)
+        #set text(1.1em, weight: "bold")
+        #smallcaps(it)
+      ]
+    
+    }
+  }
+  show heading.where(level: 2): it => block(width: 100%)[
+    #set align(center)
+    #set text(1.1em, weight: "bold")
+    #smallcaps(it)
+  ]
+  show heading.where(level: 3): it => block(width: 100%)[
+    #set align(left)
+    #set text(1em, weight: "bold")
+    #smallcaps(it)
+  ]
+
+  // Title page
+  set align(center)
+  
+  block[
+    #let jb = linebreak(justify: true)
+  
+    #text(1.5em, weight: "bold", affiliation.university) #jb
+    #text(1.2em, [
+      #smallcaps(affiliation.school) #jb
+      #affiliation.degree #jb
+    ])
+  ]
+
+  v(3fr)
+  if logo != none {
+    logo
+  }
+  v(3fr)
+
+  text(1.5em, subtitle)
+  v(1fr, weak: true)
+  text(2em, weight: 700, title)
+
+  v(4fr)
+
+  grid(
+    columns: 2,
+    align: left,
+    grid.cell(
+      inset: (right: 40pt)
+    )[
+      #if lang == "en" {
+        smallcaps("supervisor")
+      } else {
+        smallcaps("relatore")
+      }\
+      *#supervisor*
+
+      #if co-supervisor != none {
+        if lang == "en" {
+          smallcaps("co-supervisor")
+        } else {
+          smallcaps("correlatore")
+        }
+        linebreak()
+        co-supervisor.map(it => [
+          *#it*
+        ]).join(linebreak())
+      }
+    ],
+    grid.cell(
+      inset: (left: 40pt)
+    )[
+      \ \ \
+
+      #if lang == "en" {
+        smallcaps("candidate")
+      } else {
+        smallcaps("candidato")
+      }\
+      *#candidate.name* \
+      #candidate.matricola
+    ]
+  )
+
+  v(5fr)
+
+  text(1.2em, [
+    #if lang == "en" {
+      "Academic Year "
+    } else {
+      "Anno Accademico "
+    }
+    #academic-year
+  ])
+
+  pagebreak(to: "odd")
+  set par(justify: true, first-line-indent: 1em)
+  set align(center + horizon)
+
+  // Declaration of originality, prints in English or Italian
+  // depending on the `lang` parameter
+  heading(
+    level: 2,
+    numbering: none,
+    outlined: false,
+    if lang == "en" {
+      "Declaration of Originality"
+    } else {
+      "Dichiarazione di Originalità"
+    }
+  )
+  text(style: "italic", declaration-of-originality.at(lang))
+
+  pagebreak(weak: true)
+
+  // Acknowledgments
+  if acknowledgments {
+    heading(
+      level: 2,
+      numbering: none,
+      outlined: false,
+      if lang == "en" {
+        "Acknowledgments"
+      } else {
+        "Ringraziamenti"
+      }
+    )
+    include "acknowledgments.typ"
```

Comment:
This here and the abstract below has the same problem where it's searching in the package rather than the project directory. You can reproduce the problem by storing the package as a [local package](https://github.com/typst/packages?tab=readme-ov-file#local-packages) and then scaffolding it with `typst init`.

If those things differ by user, you would need to pass them as a parameter (and a user can then write `acknowledgments: include "acknowledgments.typ"`.

If they don't differ, you can move them from the template directory into the package itself.

Reply:
Sorry, I didn't notice, I'll probably delete them then and take their content as parameter, I'll check if there are no other similar problems